src/components/SubStepForms/FullNameStep.tsx

```diff
@@ -41,7 +41,14 @@ type FullNameStepProps<TFormID extends keyof OnyxFormValuesMapping> = SubStepPro
         lastName: string;
     };
 
+    /** Should show the help link or not */
     shouldShowHelpLinks?: boolean;
+
+    /** Custom label of the first input  */
```

Comment:
```suggestion
    /** Custom label of the first name input  */
```

Reply:
Mmmmmmm maybe not needed

Reply:
ya actually i like this better b/c maybe eventually the order will change / we'll add more, so let's just specify that this is for the "first name"